mindjunkies/forums/views.py

```
@@ -40,15 +44,15 @@ def post(self, request, *args, **kwargs):
         # Get course_slug from URL
         course_slug = self.kwargs.get('course_slug', '')
         course = get_object_or_404(Course, slug=course_slug)
-        
+
         # Check if this is a topic submission or a reply submission
         if 'topic_id' in request.POST:
             # This is a reply submission
             return self.handle_reply_submission(request, course_slug)
         else:
```

Ruff (0.8.2)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)